tests/e2e/util.go

```diff
@@ -6973,3 +6974,45 @@ func getHostMoref4K8sNode(
 	vmIp2MoRefMap := vmIpToMoRefMap(ctx)
 	return vmIp2MoRefMap[getK8sNodeIP(node)]
 }
+
+func setQuota(ctx context.Context, restClientConfig *rest.Config,
+	scName string, namespace string, quota string) {
+	cnsOperatorClient, err := k8s.NewClientForGroup(ctx, restClientConfig, cnsoperatorv1alpha1.GroupName)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	spq := &storagepolicyv1alpha1.StoragePolicyQuota{}
+	err = cnsOperatorClient.Get(ctx,
+		pkgtypes.NamespacedName{Name: scName + storagePolicyQuota, Namespace: namespace}, spq)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	spq.Spec.Limit.Reset()
+	spq.Spec.Limit.Add(resource.MustParse(quota))
+	framework.Logf("set quota %s", quota)
+
+	err = cnsOperatorClient.Update(ctx, spq)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+}
+
+func removeQuota(ctx context.Context, restClientConfig *rest.Config,
```

Comment:
looks like resetting the storage policy quota, rename it to resetStoragePolicyQuota

Reply:
Updated as removeStoragePolicyQuota